spec/services/reporting/total_user_count_report_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+require 'csv'
+
+RSpec.describe Reporting::TotalUserCountReport do
+  let(:report_date) { Date.new(2021, 3, 1) }
+  let(:expected_report) do
+    [
+      ['All-time user count'],
+      [expected_count],
+    ]
+  end
+
+  subject(:report) { described_class.new(report_date) }
+
+  before { travel_to report_date }
+
+  shared_examples 'a report with that user counted' do
+    let(:expected_count) { 1 }
+    it 'includes that user in the count' do
+      expect(subject.total_user_count_report).to eq expected_report
+    end
+  end
```

Comment:
This is not a big savings, but IMHO it better expresses the intent vs. repeating `expect(subject.total_user_count_report).to eq expected_report` a bunch of times.